cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -1901,6 +1899,15 @@ private void writeToStructuredTableWithPrimaryKeys(
     return fields;
   }
 
+  private List<Field<?>> getRunRecordPeerPrefix(String status, @Nullable String peerName) {
```

Comment:
where is this method called?

Reply:
removed this method since it's not being used